src/abstractdataframe/show.jl

```diff
@@ -61,10 +55,10 @@ end
 #' ourshowcompact(STDOUT, "abc")
 #' ourshowcompact(STDOUT, 10000)
 ourshowcompact(io::IO, x::Any) = showcompact(io, x) # -> Void
-ourshowcompact(io::IO, x::AbstractString) = print(io, x) # -> Void
-ourshowcompact(io::IO, x::Symbol) = print(io, x) # -> Void
+ourshowcompact(io::IO, x::AbstractString) = escape_string(io, x, "") # -> Void
+ourshowcompact(io::IO, x::Symbol) = ourshowcompact(io, string(x)) # -> Void
 ourshowcompact(io::IO, x::CategoricalValue{<:AbstractString}) =
```

Comment:
`CategoricalValue <: AbstractString` in the current master, so I could as well tag a release so that we can depend on it and remove this method.

Reply:
See https://github.com/JuliaLang/METADATA.jl/pull/11290.

Reply:
I understand we wait till CategoricalValue is merged? I have implemented what you have requested but the changes will fail on CI now.

Reply:
Yes, hopefully that won't take too much time.

Reply:
Release has been tagged now.

Reply:
Woops, making `CategoricalValue{T} <: AbstractString` creates issues when `T` is not an `AbstractString` (e.g. here it's a `Char`). The problem is that dispatch uses a method which works only for strings, and it fails later because the data isn't actually a string. We could fix this particular case, but this sounds like a fragile approach in general. Maybe we'd better restrict `T <: AbstractString` for `CategoricalArray{T}`. Let's discuss this at https://github.com/JuliaData/CategoricalArrays.jl/pull/77.

Reply:
The easiest solution for now is probably to stop creating a `CategoricalArray{Char}` and use a standard array instead.

Reply:
fixed